### PR TITLE
Fix addons modules not being installed in CI

### PR DESCRIPTION
**Pull request by atomiix**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Because of a curl settings associated to the CI context, the request to get addons modules was not successful. This PR fixes it
| Type?             | bug fix
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | CI should pass
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24586)

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -107,6 +107,9 @@ csa_guzzle:
                     timeout: "5.0"
                 headers:
                     Accept: "application/json"
+                curl:
+                    'forbid_reuse': true
+                    'fresh_connect': true
```
Comment (Progi1984) [inline app/config/config.yml]:
```suggestion
                    forbid_reuse: true
                    fresh_connect: true
```

**Review by Progi1984 — APPROVED** [review]

File: app/config/config.yml [hunk 2]
```diff
@@ -107,6 +107,9 @@ csa_guzzle:
                     timeout: "5.0"
                 headers:
                     Accept: "application/json"
+                curl:
```
Comment (PierreRambaud) [inline app/config/config.yml:110]: What a simple fix :scream:

**Review by PierreRambaud — APPROVED** [review]

File: app/config/config.yml (hunk 2 above)
Comment (matks) [reply app/config/config.yml:110]: 🤯

**Review by matks — APPROVED** [review]

**Comment by Progi1984** [comment]
@PrestaShop/qa-functional Does it need some QA for this ?

**Comment by Robin-Fischer-PS** [comment]
Hi @Progi1984 !

I don't think QA is necessary, if CI is OK.

Thanks :)

**Comment by Progi1984** [comment]
Thanks @atomiix & @Robin-Fischer-PS

**Pull request merged**